ui-manchette/src/types.ts

```diff
@@ -10,7 +10,13 @@ export type Waypoint = {
 export type InteractiveWaypoint = Waypoint & {
   styles?: CSSProperties;
   display?: boolean;
-  onClick?: (opId: string, opRef: HTMLDivElement | null) => void;
+  onClick?: (waypointId: string, waypointRef: React.RefObject<HTMLDivElement>) => void;
```

Comment:
Note to self: I'm a bit sad to have the caller take care of figuring out whether the waypoint is visible or not, it would be nicer API. Plus the underlying `HTMLDivElement` passed here to the callback can become stale while the menu is displayed (React can choose to re-render it in another DOM element any time).

 But IIRC this causes other Issues™.

Reply:
As discussed, gonna keep the menu visibility in osrd-ui and the caller will only be responsible to pass the container ref to the manchette if he want's to have the hiding behavior.